chrome/content/zoteroocr.js

```diff
@@ -6,6 +6,22 @@ Components.utils.import("resource://gre/modules/osfile.jsm");
 
 Zotero.OCR = new function() {
 
+	this.openPreferenceWindow = function(paneID, action) {
+		var io = {pane: paneID, action: action};
+		window.openDialog(
+				'chrome://zoteroocr/content/preferences.xul',
```

Comment:
If I replace this with Zotero's preferences 'chrome://zotero/content/preferences/preferences.xul' then everything works fine.

But also if I type `window.openDialog('chrome://zoteroocr/content/preferences.xul')` into the "Run JavaScript" window, this works fine even for multiple times in a row.

This is now really confusing...